app/models/project.rb

```diff
@@ -42,6 +42,7 @@ class Project < ActiveRecord::Base
   has_many :project_partners
   has_many :subgoals, -> { order 'value DESC' }
   has_many :subscriptions
+  has_many :subscription_reports
```

Comment:
seria interessante adicionado `:dependent  destroy ou restrict` nos has_many, pros dados do outro lado não ficarem perdidos quando este for deletado.